wp-includes/widgets/class-wp-widget-media.php

```diff
@@ -53,11 +55,16 @@ public function __construct( $id_base, $name, $widget_options = array(), $contro
 		$control_opts = wp_parse_args( $control_options, array() );
 
 		$l10n_defaults = array(
-			'no_media_selected' => __( 'No media selected' ),
-			'edit_media' => __( 'Edit Media' ),
+			'add_to_widget' => __( 'Add to Widget' ),
 			'change_media' => __( 'Change Media' ),
+			'edit_media' => __( 'Edit Media' ),
+			'error' => sprintf(
```

Comment:
There could be additional errors in the future, so I think this should be `attachment_missing_error` instead.